README.md

```diff
@@ -26,8 +26,9 @@ If you would like to contribute another port (which would be awesome!), please o
 ## How to get started
 ### In the browser
 ```HTML
-<script type="text/javascript" src="VerbalExpressions.js"></script>
+<script src="VerbalExpressions.js"></script>
 ```
+Or use the [jsDelivr CDN](http://www.jsdelivr.com/projects/jsverbalexpressions).
```

Comment:
please also add a code block that we can use to copy/paste the script on the page
`<script src="https://cdn.jsdelivr.net/jsverbalexpressions/0.2.0/verbalexpressions.min.js"></script>`